leftwm-core/src/utils/command_pipe.rs

```diff
@@ -287,18 +288,16 @@ fn build_move_window_to_previous_tag(raw: &str) -> Result<Command, Box<dyn std::
     Ok(Command::MoveWindowToPreviousTag { follow })
 }
 
-fn build_increase_main_width(raw: &str) -> Result<Command, Box<dyn std::error::Error>> {
-    let headless = without_head(raw, "IncreaseMainWidth ");
-    let mut parts = headless.split(' ');
-    let change: i8 = parts.next().ok_or("missing argument change")?.parse()?;
-    Ok(Command::IncreaseMainWidth(change))
+fn build_increase_main_size(raw: &str) -> Result<Command, Box<dyn std::error::Error>> {
+    let mut parts = raw.split(' ');
+    let change: i32 = parts.next().ok_or("missing argument change")?.parse()?;
+    Ok(Command::IncreaseMainSize(change))
 }
 
-fn build_decrease_main_width(raw: &str) -> Result<Command, Box<dyn std::error::Error>> {
-    let headless = without_head(raw, "DecreaseMainWidth ");
-    let mut parts = headless.split(' ');
-    let change: i8 = parts.next().ok_or("missing argument change")?.parse()?;
-    Ok(Command::DecreaseMainWidth(change))
+fn build_decrease_main_size(raw: &str) -> Result<Command, Box<dyn std::error::Error>> {
+    let mut parts = raw.split(' ');
+    let change: i32 = parts.next().ok_or("missing argument change")?.parse()?;
+    Ok(Command::DecreaseMainSize(change))
```

Comment:
*Note: The call to `without_head` was unnecessary because the `raw` parameter doesn't contain the command, it's already split out before this method is called.*